akka-stream/src/main/scala/akka/stream/scaladsl/Sink.scala

```diff
@@ -342,4 +338,6 @@ object Sink {
   def queue[T](): Sink[T, SinkQueue[T]] =
     Sink.fromGraph(new QueueSink())
 
+  def monitored[T, M](inner: Sink[T, M]): Sink[T, (M, FlowMonitor[T])] =
```

Comment:
Actually, I don`t think it should be on `Sink` as `flow.to(sink)` becomes a sink.